enable/qt/base_window.py

```diff
@@ -461,12 +461,17 @@ def _create_mouse_event(self, event):
                 y = event.position().y()
             modifiers = event.modifiers()
             buttons = event.buttons()
+        # The AttributeError is usually trigged when the mouse pointer
+        # leaves the ui window since the event of leaving the window
+        # is a "QEvent", which doesn't contain x and y positions like
+        # QMouseEvent (for mouse movement) or QEnterEvent (for mouse
+        # entering the window) from QGui.
         except AttributeError:
             pos = self.control.mapFromGlobal(QtGui.QCursor.pos())
             x = pos.x()
             y = pos.y()
-            modifiers = 0
-            buttons = 0
+            modifiers = QtCore.Qt.KeyboardModifier.NoModifier
+            buttons = QtCore.Qt.NoButton
```

Comment:
Probably should be:
```suggestion
            buttons = QtCore.Qt.MouseButton.NoButton
```
or something similar.